code/API_definitions/CAMARA/number_verification.yaml

```diff
@@ -158,6 +215,21 @@ components:
           description: Hashed phone number. SHA-256 (in hexadecimal representation) of the mobile phone number in 'E164 with +' format
           type: string
           example: 32f67ab4e4312618b09cd23ed8ce41b13e095fe52b73b2e8da8ef49830e50dba
+    ueId:
+      type: object
+      properties:
+        msisdn:
```

Comment:
@bigludo7 having in mind that in this API we haven't used MSISDN, do you think we should change it to phone_number to be aligned with the other endpoints or we keep it this way?

Reply:
Also, phone_number must be required, don't you think so?